components/patient-access-rules/ui/src/main/resources/PhenoTips/PatientAccessRightsManagement.xml

```diff
@@ -94,12 +94,36 @@
   #set ($collaborators = $request.getParameterValues('collaborator'))
   #set ($accessLevels = $request.getParameterValues('accessLevel'))
   #set ($owner = "$!{request.get('owner')}")
+  #if ("$!{owner}" != "")
+    ##check the validity of the new owner
+    #set($ownerDoc = $services.model.resolveDocument($owner))
+    #set($ownerDocRef = $services.model.resolveDocument("$owner"))
+    #if ($xwiki.getDocument($ownerDocRef).getObject("PhenoTips.PhenoTipsGroupClass"))
+      #set($validGroup = true)
+      ##check to see if user has permission to transfer ownership
+      #set ($adminGroup = $owner + " Administrators")
+      #if (!$xwiki.getUser().isUserInGroup($owner) &amp;&amp; !$xwiki.getUser().isUserInGroup($adminGroup) &amp;&amp; !$hasAdmin)
+        #set ($cannotTransfer = true)
+      #end
+    #elseif ($services.query.xwql("from doc.object(XWiki.XWikiUsers) as user where user.name = :ownerName").bindValue("ownerName", "$owner").execute() != "[]")
```

Comment:
Weird check... Instead of running the query, getting the documents, and comparing with an empty list, use `.count() > 0`.